controllers/clustergroupupgrade_controller.go

```diff
@@ -1395,27 +1467,35 @@ func (r *ClusterGroupUpgradeReconciler) getCopiedPolicies(ctx context.Context, c
 	return policiesList, nil
 }
 
-func (r *ClusterGroupUpgradeReconciler) reconcileResources(ctx context.Context, clusterGroupUpgrade *ranv1alpha1.ClusterGroupUpgrade, managedPoliciesPresent []*unstructured.Unstructured) error {
+func (r *ClusterGroupUpgradeReconciler) reconcileResources(ctx context.Context, clusterGroupUpgrade *ranv1alpha1.ClusterGroupUpgrade, managedPoliciesPresent []*unstructured.Unstructured) (bool, error) {
 	// Reconcile resources
+	isPolicyErr := false
 	for _, managedPolicy := range managedPoliciesPresent {
 
 		policyName, err := r.copyManagedInformPolicy(ctx, clusterGroupUpgrade, managedPolicy)
 		if err != nil {
-			return err
+			if _, ok := err.(*utils.PolicyErr); ok {
+				// If it's a policy error(i.e. unsupported hub template),
+				// break the loop to execute updateChildResourceNamesInStatus
+				// to update the CGU status with already created policies
+				isPolicyErr = true
+				break
+			}
+			return isPolicyErr, err
```

Comment:
Would it more clear if we just do `return false, err`? Same for the next two returns

Reply:
done